sdk/src/main/java/openconsensus/sdk/trace/internal/CurrentSpanUtils.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2019, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.sdk.trace.internal;
+
+import io.grpc.Context;
+import java.util.concurrent.Callable;
+import openconsensus.context.Scope;
+import openconsensus.trace.Span;
+import openconsensus.trace.Status;
+import openconsensus.trace.unsafe.ContextUtils;
+
+/** Util methods/functionality to interact with the {@link Span} in the {@link io.grpc.Context}. */
+public final class CurrentSpanUtils {
+  // No instance of this class.
+  private CurrentSpanUtils() {}
+
+  /**
+   * Returns The {@link Span} from the current context.
+   *
+   * @return The {@code Span} from the current context.
+   */
+  public static Span getCurrentSpan() {
+    return ContextUtils.getValue(Context.current());
+  }
+
+  /**
+   * Enters the scope of code where the given {@link Span} is in the current context, and returns an
+   * object that represents that scope. The scope is exited when the returned object is closed.
+   *
+   * <p>Supports try-with-resource idiom.
+   *
+   * @param span The {@code Span} to be set to the current context.
+   * @param endSpan if {@code true} the returned {@code Scope} will close the {@code Span}.
+   * @return An object that defines a scope where the given {@code Span} is set to the current
+   *     context.
+   */
+  public static Scope withSpan(Span span, boolean endSpan) {
```

Comment:
This method was removed from the API, and I'm wondering if we could remove it here as well.

Reply:
(and same for overloads taking a `endSpan` param)

Reply:
I removed the endSpan part. Curious if you suggest to remove the "withSpan" overall.